beetsplug/audible.py

```diff
@@ -3,19 +3,155 @@
 import pathlib
 import re
 import urllib.error
+from copy import deepcopy
 from tempfile import NamedTemporaryFile
+from typing import Dict, Iterable, List, Optional, Tuple
 
+import beets.autotag.hooks
+import Levenshtein
 import mediafile
 import yaml
 from beets import importer, util
 from beets.autotag.hooks import AlbumInfo, TrackInfo
-from beets.plugins import BeetsPlugin, get_distance
-from natsort import os_sorted
+from beets.library import Item
+from beets.plugins import BeetsPlugin
+from natsort import natsorted
 
 from .api import get_book_info, make_request, search_audible
 from .goodreads import get_original_date
 
 
+def sort_items(items: List[Item]):
+    naturally_sorted_items = natsorted(items, key=lambda i: i.title)
+    return naturally_sorted_items
+
+
+def get_common_data_attributes(track: TrackInfo) -> Dict:
+    common_track_attributes = dict(track)
+    del common_track_attributes["index"]
+    del common_track_attributes["length"]
+    del common_track_attributes["title"]
+    return common_track_attributes
+
+
+def convert_items_to_trackinfo(items: List[Item], common_attrs: Dict) -> List[TrackInfo]:
+    out = []
+    for i, item in enumerate(items, start=1):
```

Comment:
Same question here, why is the 0th item excluded?

Reply:
The 0th item is not excluded, this just makes it so that i begins at 1. You can think of it as making the given list be indexed to 1 instead of 0.